packages/123done/static/index.html

```diff
@@ -9,12 +9,12 @@
     <meta name="firefox-accounts" content="supported" />
     <link
       rel="stylesheet"
-      href="/bower_components/normalize-css/normalize.css"
+      href="/components/normalize.css"
```

Comment:
I remove bower because we _really_ shouldn't be using it anymore. Instead of adding a build step I just added the files to git. This is 123done people, don't judge me.